src/Tizen.NUI/src/internal/Interop/Interop.DoGestureSignal.cs

```diff
@@ -49,10 +49,10 @@ internal static partial class DoGestureSignal
             public static extern uint GetSizeOfGestureInfo();
 
             [global::System.Runtime.InteropServices.DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_Signal_GesturePairToVoid_GetResult")]
-            public static extern int GetResult(global::System.IntPtr jarg1);
+            public static extern bool GetResult(global::System.IntPtr jarg1);
 
             [global::System.Runtime.InteropServices.DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_Signal_GesturePairToVoid_SetResult")]
-            public static extern void SetResult(global::System.IntPtr jarg1, int result);
+            public static extern void SetResult(global::System.IntPtr jarg1, bool result);
```

Comment:
The signature is now inconsistent with `CSharp_Dali_Signal_GesturePairToVoid_SetResult` here: https://review.tizen.org/gerrit/#/c/platform/core/uifw/dali-csharp-binder/+/259004/3/dali-csharp-binder/src/signal_wrap.cpp (`bool` vs. `int`).

Reply:
Thank you for your review.
I've just followed other C# bindings, but they were remainders mistakenly bounding by auto binder in the past.. :(
I changed csharp-binder side.